Whitespace cleanup

waldyrious opened the pull request:
Cleaning up whitespace issues identified while working on #45.

FlominatorTM commented:
Worried about the translations coming from translatewiki. In theory changing the master should be sufficient

waldyrious commented:
Either way, I'm going to mark this PR as a draft for the time being, since I found other whitespace issues that I'm working through locally.

waldyrious commented:
> In theory changing the master should be sufficient

I'm not sure what you mean. The whitespace in `wikiblame_inc/en.php` already contains the fix that is now being applied to the translation files. Or do you mean something else by "master"?

In any case, the change in the language files are almost all the space after the `*`; I will split those changes to a separate commit to make the diffs less unwieldy.

waldyrious commented:
Ok, I've split the changes in the language files into a separate commit (b337426), and added the additional fixes I mentioned above to the first commit. It's probably helpful to review the diffs of each commit separately

I'd also recommend checking the first commit in the [hide whitespace changes](https://github.com/FlominatorTM/wikiblame/pull/46/commits/40f2a451490098ae34fb282fdd7b9f69f977113e?w=1) mode of GitHub's interface. As you can see, the only changes that that view shows are the brackets moved to their own lines, the blank lines that were removed entirely, and the double semicolons that were collapsed to single ones.

tacsipacsi commented:
Looking at the [Translatewiki source code](https://github.com/wikimedia/mediawiki-extensions-Translate/blob/0e36f66dfca92f6427769be7c2e56e6b1f46c8bf/ffs/FlatPhpFFS.php), it doesn’t _modify_ the PHP files, but rather rewrites them, keeping only the `@author` PHPDoc tags. This means that the next time the translations are updated, the spaces will be re-added. You could upload a Gerrit change in the Translate repo to avoid generating the the spaces, but I don’t think it’s worth it.
